lib/transform.go

```diff
@@ -35,7 +35,7 @@ type ApplyParams struct {
 	Wait      bool
 
 	Source       string
-	ScriptOutput io.Writer
+	ScriptOutput io.Writer `json:"-"`
```

Comment:
I wonder if it would be not too hard to wire up an io.Writer that forwards output to the websocket?